rcongui_public/src/hooks/use-theme-provider.tsx

```diff
@@ -1,3 +1,5 @@
+"use client"
```

Comment:
is that really needed? We do not have server-side rendering anywhere, aren't we? 🤔

Reply:
No it is not. Brain seizure

Reply:
Once removed, I merge 🥳